src/ut_internal.h

```diff
@@ -30,7 +30,7 @@
 
 #include <string.h>
 
-#define UT_MAX_FILENAME_STRING_SIZE (32)
+#define UT_MAX_FILENAME_STRING_SIZE (256)
```

Comment:
where is this used, and why would we not be defining it in the external header file rather than the internal one?

Reply:
I have reverted this change.